includes/helm.py

```diff
@@ -241,74 +241,53 @@ def get_info_from_helm(component, repo, services):
               )
             else:  # default either to false or None
               update_dict(helm_envs, env, {mod_security_type[0]: mod_security_type[1]})
+        if not am.isDataUnavailable():
```

Comment:
you could invert this check on the alertmanager class and `if am.isDataAvailable()`

(I didn't know python syntax to invert it 😅 )